external/hedley/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+if(NOT TARGET hedley)
+    add_library(hedley INTERFACE IMPORTED GLOBAL)
+    target_include_directories(hedley INTERFACE "${CMAKE_CURRENT_LIST_DIR}/include")
```

Comment:
Fails to build with CMake 3.10. Proposed change:
```diff
-    target_include_directories(hedley INTERFACE "${CMAKE_CURRENT_LIST_DIR}/include")
+    set_target_properties(hedley PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_LIST_DIR}/include")
```
Also `No newline at end of file` - not an issue, but would make `cat` for example not move the console prompt at beginning of next line.